### PR TITLE
fix: reading callback url from env

### PR DESCRIPTION
**Pull request by amosmachora**
I am now reading the callback url from environment variables to fix the issue raised by @ojwanganto .

### CODE REVIEW

**Review by ojwanganto — APPROVED** [review]
LGTM

File: app/api/mpesa/stk-push/route.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ export const POST = async (request: NextRequest) => {
   const requestBody: RequestBody = {
     accountReference,
     amount,
-    callbackURL: `https://billing.kenyahmis.org/api/mpesa/stk-push-callback`,
+    callbackURL: `${process.env.MPESA_APP_BASE_URL}/api/mpesa/stk-push-callback`,
```
Comment (ojwanganto) [inline app/api/mpesa/stk-push/route.ts:21]: Do we have a way to check for null if the API is not configured?

Comment (amosmachora) [reply app/api/mpesa/stk-push/route.ts:21]: Actually thats a great observation. Its not safe to assume. I will add the functionality to make sure the build process fails if environment vars are missing.

Comment (ojwanganto) [reply app/api/mpesa/stk-push/route.ts:21]: Great

**Pull request merged**